custom_components/localtuya/__init__.py

```diff
@@ -189,8 +189,10 @@ def update(self):
             self._status = self._device.status()
             self.status_updated()
         except Exception:
+            _LOGGER.debug("update hit exception, setting self._available=false")
```

Comment:
We need Somme way to de with this, but I don't think this is the right way. It is probably better that we log an exception when things go south but only do it once until the connection is restored. I can sketch something up. Same for the debug below.

Reply:
Ok, i'll just remove these debug lines for now